OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - anishasthana
+  - VaishnaviHire
```

Comment:
I'm adding Vaishnavi as an approver in the short term until we've figured out who else will be contributing to Model Mesh